src/shell/info.c

```diff
@@ -175,6 +175,9 @@ static int shell_init_jobinfo (flux_shell_t *shell, struct shell_info *info)
         shell_log_error ("error parsing jobspec: %s", error.text);
         goto out;
     }
+    if (strstarts (error.text, "Unsupported jobspec version")) {
+        shell_warn ("warning parsing jobspec: %s", error.text);
+    }
```

Comment:
`error.text` could be uninitialized here I think?  It might be better to just eliminate the version check in `jobspec_parse()` and move the test here:
```c
if (jobspec->versions != 1)
    // warn
```